src/production/secrets/maevsi/recaptcha-key.secret.template

```diff
@@ -0,0 +1 @@
+<key>
```

Comment:
This file is not used right now because a file containing the directory of a file containing the secret can't be given to the nuxt turnstile widget. I'll tweak the code and make a PR for this issue.

Reply:
WIP: https://github.com/nuxt-modules/turnstile/pull/174

Reply:
Turnstile v0.2.0 is released, the filename can now be changed :partying_face:

Reply:
I implemented the file for turnstile